Makefile

```diff
@@ -74,13 +76,25 @@ ifneq ($(MM_SERVICESETTINGS_ENABLEDEVELOPER),)
 	cd server && env CGO_ENABLED=0 $(GO) build $(GO_BUILD_FLAGS) $(GO_BUILD_GCFLAGS) -trimpath -o dist/plugin-$(DEFAULT_GOOS)-$(DEFAULT_GOARCH);
 else
 	cd server && env CGO_ENABLED=0 GOOS=linux GOARCH=amd64 $(GO) build $(GO_BUILD_FLAGS) $(GO_BUILD_GCFLAGS) -trimpath -o dist/plugin-linux-amd64;
-	cd server && env CGO_ENABLED=0 GOOS=linux GOARCH=arm64 $(GO) build $(GO_BUILD_FLAGS) $(GO_BUILD_GCFLAGS) -trimpath -o dist/plugin-linux-arm64;
-	cd server && env CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 $(GO) build $(GO_BUILD_FLAGS) $(GO_BUILD_GCFLAGS) -trimpath -o dist/plugin-darwin-amd64;
-	cd server && env CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 $(GO) build $(GO_BUILD_FLAGS) $(GO_BUILD_GCFLAGS) -trimpath -o dist/plugin-darwin-arm64;
-	cd server && env CGO_ENABLED=0 GOOS=windows GOARCH=amd64 $(GO) build $(GO_BUILD_FLAGS) $(GO_BUILD_GCFLAGS) -trimpath -o dist/plugin-windows-amd64.exe;
+##  only AMD64 Linux is supported for now.
+##	cd server && env CGO_ENABLED=0 GOOS=linux GOARCH=arm64 $(GO) build $(GO_BUILD_FLAGS) $(GO_BUILD_GCFLAGS) -trimpath -o dist/plugin-linux-arm64;
+##	cd server && env CGO_ENABLED=0 GOOS=darwin GOARCH=amd64 $(GO) build $(GO_BUILD_FLAGS) $(GO_BUILD_GCFLAGS) -trimpath -o dist/plugin-darwin-amd64;
+##	cd server && env CGO_ENABLED=0 GOOS=darwin GOARCH=arm64 $(GO) build $(GO_BUILD_FLAGS) $(GO_BUILD_GCFLAGS) -trimpath -o dist/plugin-darwin-arm64;
+##	cd server && env CGO_ENABLED=0 GOOS=windows GOARCH=amd64 $(GO) build $(GO_BUILD_FLAGS) $(GO_BUILD_GCFLAGS) -trimpath -o dist/plugin-windows-amd64.exe;
```

Comment:
I think this will make it so dev machines can't run this except for `linux-amd64`. Maybe handling this should be in a separate PR

Reply:
Dev machines will build via the MM_SERVICESETTINGS_ENABLEDEVELOPER which builds the current architecuture.

Reply:
If I don't comment these out here, the resulting tarball includes all the binaries and the plugin is 75MB

Reply:
@wiggin77 In practice many people don't have that env var enabled. It's common to have the server configured with developer mode, but in this case the env of the specific terminal compiling the plugin needs to have this env var set. I think we should generally assume that this env var is not set

Reply:
@mickmister  What do you suggest?  We cannot have all these non-supported architectures in the release.

Reply:
@wiggin77 We can add an env var like `OFFICIAL_PLUGIN_RELEASE: true` in this workflow: https://github.com/mattermost/delivery-platform/blob/2332ed4f90470fd09a10c7f0677b31763712e919/.github/workflows/release-mattermost-plugins.yaml#L141

Which in turn calls this `plugin-ci` workflow that runs `make dist` https://github.com/mattermost/actions/blob/main/plugin-ci/build/action.yaml

Then the plugin repo's `Makefile` can conditionally choose which architectures should be built based on the value of the `OFFICIAL_PLUGIN_RELEASE` env var. I think setting this env var in the first workflow makes sense, since we're absolutely sure a release is being built in that context.

Reply:
@mickmister that sounds great, but I need a build today.  I'm not concerned with Mac/Windows support, even on dev machines at this point.  They can set a env var locally as a workaround for now.

Reply:
Alternatively, we can make it so `make dist` is only used by CI for this plugin or something like that. Either way, I think some assumption on the side of CI or the plugin needs to be changed as far as how to handle compiling the release

Reply:
@mickmister I have made the change only for CI.  CI will build only Linux AMD64.